wgsl/index.bs

```diff
@@ -40,10 +40,14 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
+ * Text based language which is writable and readable by programmers
+ * [SHORTNAME] is as small as possible to represent what is needed
+ * Strong static typing
+ * Directly represents structured control flow
+ * Enables simple and efficient compilation to SPIR-V, Metal Shading Language, and HLSL
+     * No polymorphism, no general pointers, no overloads, etc
+ * Enables simple and efficient conversion from SPIR-V to [SHORTNAME] to SPIR-V while preserving the performance properties of the original shader
  * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
  * Each item in this spec *must* provide the mapping to SPIR-V for the construct
```

Comment:
We should probably join these two together into a single item. What's the difference between "constructs" and "items?"

It also seems that they aren't really goals, but instead are a means to achieving the "simple and efficient conversion from SPIR-V." These two probably belong in the README instead.

Reply:
In my mind these two are different to the simple and efficient conversion goal.  The key part here is that the semantics of WGSL are inherited from SPIR-V.  This is why we can have confidence that WGSL works as a shader language and doesn't specify things that are inherently hard or ambiguous across multiple GPU architectures.  This is how WGSL leverages all the work that has been done to ensure this in SPIR-V.

Reply:
I'm in agreement with @damyanp here. The semantics of SPIR-V are an important goal for WGSL.